docs/middlewares.md

```diff
@@ -237,3 +237,26 @@ handler(event, {}, (_, body) => {
   })
 })
 ```
+
+## [doNotWaitForEmptyEventLoop](/src/middlewares/doNotWaitForEmptyEventLoop.js)
+
+Sets context.callbackWaitsForEmptyEventLoop property to false. This will prevent lambda for timing out because of hanging connection
```

Comment:
please put context.callbackWaitsForEmptyEventLoop and false in backticks so that they will look like this:

`context.callbackWaitsForEmptyEventLoop`

`false`

Reply:
nice spot